.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "node"
+  - "7"
+  - "6"
+  - "5"
```

Comment:
They checks with older versions of node as well, line 3 will check with the latest stable release.